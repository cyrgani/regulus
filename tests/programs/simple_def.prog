_(
	def(
		my_fn,
		@(x, y, z),
		print("x:", x, "y:", y, "z:", z)
	),
	my_fn(2, true, null),
	def(lambda, args(), print(null)),
	lambda()
)