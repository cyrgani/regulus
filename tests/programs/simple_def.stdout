x: 2 y: true z: null
null
